README.md

```diff
@@ -1 +1,56 @@
 # Canopy Frontend RFC
+The Canopy Frontend RFC (Request for Comments) is a git repository where the technologies and processes that apply to all
+frontend squads at Canopy are described.
+
+## Purpose
+1. Provide a way to enact change in Canopy's frontend that impacts all projects and squads.
+2. Clarify what things we are aligned on across the entire front-end, so that everything else can be decided by the members of each squad.
+
+## What this isn't
+1. An infringement on the autonomy of squads. The idea is that if we're aligned on the few important things, we can be autonomous on everything else.
+2. Something oppressively prescriptive.
+3. A technical review process where people get feedback on difficult problems they're facing. Although important, the RFC is not how
+   technical reviews happen.
+
+## Signs that this is working
+1. The content is up-to-date and people care about it.
+2. There are regular updates.
+3. Things are regularly removed.
+4. We change the process when it's annoying or doesn't work well.
+5. Ideas have a safe place to develop before being criticized.
+
+## Signs that this isn't working
+1. This blocks people from getting stuff done.
+2. Ivory tower gatekeeping.
+3. Changes are discouraged instead of encouraged.
+4. "Well that's just the way it is."
+5. The process is heavy, bureaucratic, or long.
+
+## What to do if the RFC isn't working
+Propose a change to the process!
+
+## Process for change
+**Anyone** at Canopy can propose a change.
+
+<img src="https://mk0radicalcandov3r1t.kinstacdn.com/wp-content/uploads/2017/02/gsd-wheel.png" width="300" />
+
+### Step 1 - Idea
+During the idea phase, we should encourage instead of critique. A person can use the #frontend-rfc slack channel
+to discuss, clarify, and foster an idea. We should examine with an open mind the pros and cons, options, and implementation.
+
+Another part of the idea phase is asking the question "Should this be autonomous to the squads or enforced consistently in all squads?"
+
+### Step 2 - Proposal
+A proposal is a pull request to this git repository. Here, an idea is debated and discussed by anyone who wants to participate. Reviewers can make comments
+asking for changes or they can challenge the basis of the proposal.
+
+If a pull request becomes stale, close it.
+
+### Step 3 - Approval
+Once the author and reviewers have commented and the discussion and proposal have stabilized, the proposal will be merged or closed. The process for approval
+will start out fairly loose and can be refined over time. To start out, we'll shoot for consensus among all participants and fallback to chapter leads helping
+to make the decision if consensus doesn't happen.
+
+### Step 4 - Implementation
+Once a proposal is merged, it is now part of the RFC. The changes or additions to the RFC apply to the whole frontend and we should make whatever
+changes are needed to our code, processes, and organization.
```

Comment:
Should a suggested timeline be part of a proposal or something?

What if an RFC is merged but is never actually implemented by one or many squads?

Reply:
RFC for what to do? haha. I don't know at the moment, and I think it may be best to be a group decision anyway.

Reply:
yeah i wasn't sure how specifically to try to explain this step

Reply:
Maybe it is loosely up to the chapter leads to make sure it is being adopted across squads. If it isn't, that should be brought up in another PR?